raiden-cli/package.json

```diff
@@ -31,6 +31,7 @@
     "typescript": "^4.2.4"
   },
   "dependencies": {
+    "@ethersproject/providers": "^5.1.0",
```

Comment:
Please, don´t import the subprojects from ethers on dapp/cli. This can be accessed from `ethers` instead, which is already available.